about.html

```diff
@@ -0,0 +1,170 @@
+          </p>
+          <div class="about_contact">
+            <small>
+              please contact us via Email for any further questions about FF
+              National Summits!
+            </small>
+            <p class="text_mail">ffsummits2022@ffnaija.org</p>
+            <hr class="mail-line" />
+          </div>
+        </div>
+      </div>
+
+        <!-- Programs -->
+        <article class="about-section">
+          <div class="logo-text-div">
+            <h2 class="logo-text">Fit-Fam National Summits 2022 Logo</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem ipsum dolor sit amet consectetur adipisicing elit. At
+              quisquam cupiditate nam animi veritatis eum. Molestias doloremque
+              asperiores dolorum iusto Lorem ipsum dolor sit amet consectetur adipisicing elit. Vel, excepturi!</small
+            >
+          </div>
+
+          <div class="about_logo"><img src="./img/fitfam.jpg" alt="" /></div>
+          <hr />
+
+          <div class="logo-text-div header">
+            <h2 class="logo-text">See the past F-F National summits</h2>
+            <hr class="about_hr" />
+            <small class="small_paragraph"
+              >Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quis
+              quas rem placeat deleniti unde aut sapiente nostrum, laudantium et amet.
+              delectus.</small
+            >
+          </div>
+        </article>
+
+        <article class="picture_presentation">
+          <div class="previous_events">
+            <div class="event_image">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+              <span class="conference_year year">2015</span>
+              <span class="conference_venue venue"
+                >Dominion City Ajah Lagos Nigeria</span
+              >
+            </div>
+            <div class="event_image2">
+              <img
+                src="./img/fit-women-conference-2021.jpg"
+                alt="past-event-img"
+              />
+            </div>
+          </div>
+        </article>
+      </section>
+
+      <footer>
+        <div class="footer_bg">
+          <div class="program_h2 partner_h2">
+            <h2 class="main_program">Partner</h2>
+            <hr class="program_hr" />
+          </div>
+
+          <ul class="footer_list list">
+            <li><img src="./img/dinma.jpg" alt="" /></li>
+            <li><img src="./img/food-logo.jpg" alt="" /></li>
+            <li><img src="./img/fitfam.jpg" alt="" /></li>
+          </ul>
+        </div>
+
+        <div class="footer_down">
+          <img src="./img/creative-commons-license-logo.jpg" alt="" />
+
+          <div>
+            <p>
+              2022 &copy; Anyanwu Chidinma Faith for Fit-Fam National Summits
+              some rights reserved
+            </p>
```

Comment:
In the design provided to match as much as possible, the background of the footer in the About page on the desktop version is black.

- [ ] Kindly, change the background colour of the footer to match the design;
- [ ] For mobile version, please keep the background of the footer white, to match the design provided;

Your design:
![image](https://user-images.githubusercontent.com/83596135/163587881-ca4f1981-e9f9-41f1-be9a-dbc95b7ef66e.png)

Given design:
![image](https://user-images.githubusercontent.com/83596135/163587916-1c215a10-0353-48ed-9a69-5f1f7a8ac712.png)